lib/src/values/logic_value.dart

```diff
@@ -134,10 +134,10 @@ class LogicValue {
   }
 }
 
-/// Converts a binary [String] representation to a binary [int].
+/// Converts a binary [String] representation to a binary [int]. Ignores all '_' in the provided binary.
 ///
 /// Exactly equivalent to `int.parse(s, radix:2)`, but shorter to type.
```

Comment:
We should remove part from the doc comment since it's no longer exactly equivalent.  Move the "Ignores all ..." part to where this "int.parse..." line is, so that the first line remains shorter and the additional detail of ignoring _ is separated by a blank line.